openstates/in/bills.py

```diff
@@ -36,6 +39,7 @@ def check_vote_counts(vote):
         pass
```

Comment:
Ah, I think that's actually legacy code. Sorry. I accidentally rewrote exactly this below. I will consolidate.